docs/eventing/brokers/broker-types/kafka-broker/configuring-kafka-features.md

```diff
@@ -0,0 +1,206 @@
+installation step:
+
+```yaml
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: kafka-config-logging
+  namespace: knative-eventing
+data:
+  config.xml: |
+    <configuration>
+      <appender name="jsonConsoleAppender" class="ch.qos.logback.core.ConsoleAppender">
+        <encoder class="net.logstash.logback.encoder.LogstashEncoder"/>
+      </appender>
+      <root level="INFO">
+        <appender-ref ref="jsonConsoleAppender"/>
+      </root>
+    </configuration>
+```
+
+To change the logging level to `DEBUG`, you must:
+
+1. Apply the following `kafka-config-logging` `ConfigMap` or replace `level="INFO"` with `level="DEBUG"` to the
+`ConfigMap` `kafka-config-logging`:
+
+    ```yaml
+    apiVersion: v1
+    kind: ConfigMap
+    metadata:
+      name: kafka-config-logging
+      namespace: knative-eventing
+    data:
+      config.xml: |
+        <configuration>
+          <appender name="jsonConsoleAppender" class="ch.qos.logback.core.ConsoleAppender">
+            <encoder class="net.logstash.logback.encoder.LogstashEncoder"/>
+          </appender>
+          <root level="DEBUG">
+            <appender-ref ref="jsonConsoleAppender"/>
+          </root>
+        </configuration>
+    ```
+
+2. Restart the `kafka-broker-receiver` and the `kafka-broker-dispatcher`, by entering the following commands:
+
+    ```bash
+    kubectl rollout restart deployment -n knative-eventing kafka-broker-receiver
+    kubectl rollout restart deployment -n knative-eventing kafka-broker-dispatcher
+    ```
+
+## Configuring the order of delivered events
+
+When dispatching events, the Kafka broker can be configured to support different delivery ordering guarantees.
+
+You can configure the delivery order of events using the `kafka.eventing.knative.dev/delivery.order` annotation on the `Trigger` object:
+
+```yaml
+apiVersion: eventing.knative.dev/v1
+kind: Trigger
+metadata:
+  name: my-service-trigger
+  annotations:
+     kafka.eventing.knative.dev/delivery.order: ordered
+spec:
+  broker: my-kafka-broker
+  subscriber:
+    ref:
+      apiVersion: serving.knative.dev/v1
+      kind: Service
+      name: my-service
+```
+
+The supported consumer delivery guarantees are:
+
+* `unordered`:  An unordered consumer is a non-blocking consumer that delivers messages unordered, while preserving proper offset management. Useful when there is a high demand of parallel consumption and no need for explicit ordering. One example could be processing of click analytics.
+* `ordered`: An ordered consumer is a per-partition blocking consumer that waits for a successful response from the CloudEvent subscriber before it delivers the next message of the partition. Useful when there is a need for more strict ordering or if there is a relationship or grouping between events. One example could be processing of customer orders.
+
+The `unordered` delivery is the default ordering guarantee.
```

Comment:
I'd leave these where they were and document only the `config-kafka-features` ConfigMap here.

Reply:
cc @Cali0707

Reply:
Oh whoops, thanks for the ping @pierDipi - this PR got lost in my inbox